web/src/lib/types.ts

```diff
@@ -92,3 +92,8 @@ export enum Environment {
   Production = "production",
   Staging = "staging",
 }
+
+export enum Auth0Error {
+  Auth0IdMismatch = "auth0_id_mismatch",
```

Comment:
Do we want to show more specific errors in case of failed login? At the moment these codes are not in use.
I can add more specific error messages or remove these codes.

Reply:
Removed in [1164888](https://github.com/worldcoin/developer-portal/pull/387/commits/1164888afca1b9e696ccf2030518dc58d35689c6)